misc/skins/catppuccin-latte.ini

```diff
@@ -0,0 +1,155 @@
+    rightbottom = ┘
+    topmiddle = ┬
+    bottommiddle = ┴
+    leftmiddle = ├
+    rightmiddle = ┤
+    cross = ┼
+    dhoriz = ─
+    dvert = │
+    dlefttop = ┌
+    drighttop = ┐
+    dleftbottom = └
+    drightbottom = ┘
+    dtopmiddle = ┬
+    dbottommiddle = ┴
+    dleftmiddle = ├
+    drightmiddle = ┤
+
+[core]
+    _default_ = color238;color255
+    selected = color255;color69
+    marked = color172;color255
+    markselect = color172;color69
+    gauge = color238;color250
+    input = color255;color69
+    inputunchanged = color245;color69
+    inputmark = color69;color255
+    disabled = color245;color248
+    reverse = color255;color238
+    commandlinemark = color255;color238
+    header = color172;color255
+    inputhistory =
+    commandhistory =
+    shadow = color245;color252
+
+[dialog]
+    _default_ = color255;color248
+    dfocus = color255;color69
+    dhotnormal = color69;color248
+    dhotfocus = color69;color69
+    dtitle = color69;color248
+
+[error]
+    _default_ = color238;color210
+    errdfocus = color255;color248
+    errdhotnormal = color172;color210
+    errdhotfocus = color172;color248
+    errdtitle = color172;color210
+
+[filehighlight]
+    directory = color238;
+    executable = color151;
+    symlink = color245;
+    hardlink =
+    stalelink = color210;
+    device = color219;
+    special = color69;
+    core = color210;
+    temp = color245;
+    archive = color219;
+    doc = color180;
+    source = color123;
+    media = color151;
+    graph = color123;
+    database = color210;
+
+[menu]
+    _default_ = color238;color69
+    menusel = color238;color255
+    menuhot = color172;color69
+    menuhotsel = color172;color255
+    menuinactive = color255;color69
+
+[popupmenu]
+    _default_ = color238;color69
+    menusel = color172;color255
+    menutitle = color172;color69
+
+[buttonbar]
+    hotkey = color238;color255
+    button = color255;color69
+
+[statusbar]
+    _default_ = color255;color69
+
+[help]
+    _default_ = color255;color248
+    helpitalic = color210;color248
+    helpbold = color69;color248
+    helplink = color255;color69
+    helpslink = color172;color69
+    helptitle = color69;color248
+
+[editor]
+    _default_ = color238;color255
+    editbold = color172;color151
+    editmarked = color255;color69
+    editwhitespace = color69;color255
+    editnonprintable = ;color255
+    editlinestate = color238;color69
+    bookmark = color238;color210
+    bookmarkfound = color255;color151
+    editrightmargin = color69;color255
+    editframeactive = color238;
+    editframedrag = color151;
+
+[viewer]
+    _default_ = color238;color255
+    viewbold = color172;color255
+    viewunderline = color210;color255
+    viewselected = color172;color69
+
+[diffviewer]
+    added = color238;color151
+    changedline = color69;color69
+    changednew = color210;color69
+    changed = color238;color69
+    removed = color238;color210
+    error = color210;color248
+
+[widget-panel]
```

Comment:
Historically, it is so that our 256+ color skins use "fancy" characters for rendering (see, e.g. `yadt256.ini` for an example of a "thin" skin - that is one not using the double lines) instead of normal terminal characters because it is assumed that if the terminal can display 256 colors, it also must be able to display UTF-8.

I think it would make sense to stick to this pattern for these color schemes unless you want to design something special for them.

What do you think?